mods/ra/rules/aircraft.yaml

```diff
@@ -332,7 +332,7 @@ U2:
 	Inherits: ^Plane
 	Health:
 		HP: 2000
-	EditorOnlyTooltip:
+	Tooltip:
 		Name: U2 Spy Plane
```

Comment:
The "U2" seems unnecessary / unpolished here (it is nothing like a real U2).  Can we change this to "Spy Plane" while we're at it?